src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorDataSourceDownloaderIntegrationTest.java

```diff
@@ -0,0 +1,134 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.nio.file.Path;
+
+/**
+ * Class to test the {@link FuncotatorDataSourceDownloader}.
+ * Created by jonn on 8/24/18.
+ */
+public class FuncotatorDataSourceDownloaderIntegrationTest extends CommandLineProgramTest {
+
+    //==================================================================================================================
+    // Private Static Members:
+
+    private static final boolean doDebugTests = false;
```

Comment:
`doDebugTests` -> `doFullScaleTests` (and add comment explaining why they're off by default)

Reply:
Fixed.